cmd/oras/internal/option/platform.go

```diff
@@ -25,28 +25,35 @@ import (
 	"github.com/spf13/pflag"
 )
 
+const (
+	PlatformFlagName = "platform"
+)
+
 // Platform option struct.
 type Platform struct {
-	platform string
-	Platform *ocispec.Platform
+	PlatformFlag string
+	Platform     *ocispec.Platform
 }
```

Comment:
Instead of Lookup at line 40, we can add a field named `FlagDescription` with customized description.